android/tangram/src/com/mapzen/tangram/MapController.java

```diff
@@ -738,12 +760,27 @@ public void setDebugFlag(DebugFlag flag, boolean on) {
 
     /**
      * Enqueue a scene component update with its corresponding YAML node value
-     * @param componentPath The YAML component path delimited by a '.' (example "scene.animated")
-     * @param value A YAML valid string (example "{ property: true }" or "true")
+     * @param sceneUpdate A {@code SceneUpdate}
      */
-    public void queueSceneUpdate(String componentPath, String value) {
+    public void queueSceneUpdate(SceneUpdate sceneUpdate) {
         checkPointer(mapPointer);
-        nativeQueueSceneUpdate(mapPointer, componentPath, value);
+        nativeQueueSceneUpdate(mapPointer, sceneUpdate.getPath(), sceneUpdate.getValue());
+    }
+
+    /**
+     * Enqueue a scene component update with its corresponding YAML node value
+     * @param sceneUpdates List of {@code SceneUpdate}
+     */
+    public void queueSceneUpdate(List<SceneUpdate> sceneUpdates) {
+        checkPointer(mapPointer);
```

Comment:
Hmm, tangram will be handling a `null` or `empty` `sceneUpdates` gracefully, through the jni code here: https://github.com/tangrams/tangram-es/blob/4b543616be5d1bebc1cfc25b56e4d8037ad61088/android/tangram/jni/jniExports.cpp#L470-L483.

Am I missing java thingy, which will help here?